MAINTAINERS

```diff
@@ -591,7 +598,7 @@ made through a pull request.
 
 	[people.jfrazelle]
 	Name = "Jessie Frazelle"
-	Email = "jess@docker.com"
+	Email = "jess@linux.com"
```

Comment:
what? there are tabs?? you MUST use spaces! :-)
yea - I know about how affiliations change, and its not perfect, but just thought I'd +1 what @thaJeztah said because I do know it is tracked at some level by some people.